models/standalone-price/src/builder.spec.ts

```diff
@@ -166,6 +167,11 @@ describe('builder', () => {
         active: expect.any(Boolean),
         __typename: 'StandalonePrice',
       })
-    )
-  );
+    );
+
+    expect(standalonePrice.customerGroup?.id).toBe(
+      standalonePrice.customerGroupRef?.id
+    );
+    expect(standalonePrice.channel?.id).toBe(standalonePrice.channelRef?.id);
```

Comment:
These values weren’t synchronized before due to a glitch in the transformer’s addFields function. See below